hack/demo/configs/templates/advertise.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: frrk8s.metallb.io/v1beta1
+kind: FRRConfiguration
+metadata:
+  name: advsome
```

Comment:
nit: rename these to something clearer - advertise-filtered, advertise-all, nodeselectors, ..

Reply:
done